src/utils/utils.js

```diff
@@ -339,6 +339,20 @@ const linkSocial = async (token, provider, uid, email, done) => {
     return done(null, primary_account);
 };
 
+const sendRequestToken = (reqToken, res) => {
+    const payload = {
+        request_token: reqToken,
+    };
+
+    const token = jwt.sign(payload, keys.privateKey, {
+        expiresIn: keys.reqExpTime,
+        algorithm: 'RS256',
+    });
+
+    res.send(token);
```

Comment:
Use this method just to obtain the request token not to send it via `res`. Also, rename the method to `getRequestToken` instead.